src/main/java/de/mprengemann/intellij/plugin/androidicons/model/Destination.java

```diff
@@ -0,0 +1,28 @@
+package de.mprengemann.intellij.plugin.androidicons.model;
+
+import de.mprengemann.intellij.plugin.androidicons.util.TextUtils;
+
+public enum Destination {
+    Drawable,
+    Mipmap;
+
+    private static final Destination[] DESTINATION_VALUES = new Destination[] {
+        Drawable, Mipmap
+    };
+
+    public static Destination from(String string) {
+        if (TextUtils.isEmpty(string)) {
+            throw new IllegalArgumentException();
+        }
+        if (string.equalsIgnoreCase(Drawable.toString())) {
```

Comment:
instead of going through all values, I'd prefer a loop like
```
for (Destination destination : values()) {
     if (destination.toString().equalsIgnoreCase(string)) {
          return destination;
     }
}
```

Reply:
I was trying to mimic the code style of Format enum so that it is consistent across the codebase

Reply:
Yeah I saw that, not sure why I did it back then like that. If you want you can change format as well ;)

Reply:
valueof() - case sensitive, or from() with ignore case? From the usage i have not seen anything that wouldn't work with valueOf (except for exception handling)